spek-junit-platform-engine/src/main/kotlin/org/jetbrains/spek/engine/SpekTestEngine.kt

```diff
@@ -41,7 +41,7 @@ class SpekTestEngine: HierarchicalTestEngine<SpekExecutionContext>() {
     val defaultInstanceFactory = object: InstanceFactory {
         override fun <T: Spek> create(spek: KClass<T>): T {
             return spek.objectInstance ?: spek.constructors.first { it.parameters.isEmpty() }
-                .call()
+                    .call()
```

Comment:
Minor stuff, you can ignore this. We should probably agree on the code style at some point.

Reply:
I've tried :( IntelliJ just does that, I can edit it in text editor if you want

Reply:
You just need to set your trailing indent to 4 spaces in the coding style in IDEA, I think the default is 8.